rakelib/connectivity.rake

```diff
@@ -11,6 +11,8 @@ Rails.logger = Logger.new(STDOUT)
 REDIS_CONFIG = Rails.application.config_for(:redis).freeze
 Redis.current = Redis.new(REDIS_CONFIG['redis'])
 
+require 'sentry_logging'
+require 'sentry/rescue_evss_errors'
 require 'evss/claims_service'
```

Comment:
Added these here so as to not break the rake task.

Reply:
can you remove all the requires from this file? they can all be autoloaded